services/sso_lifecycle.go

```diff
@@ -25,6 +25,7 @@ var _ = ServicesDescribe("SSO Lifecycle", func() {
 		if !Config.GetIncludeSSO() {
 			Skip(skip_messages.SkipSSOMessage)
 		}
+		apiEndpoint = Config.Protocol() + Config.GetApiEndpoint()
```

Comment:
This line seems to be unused. Can it be removed?

Reply:
Ah, so `apiEndpoint` is in fact consumed in the tests below - on [L79](https://github.com/cloudfoundry/cf-acceptance-tests/blob/ee8e82d7199fa2726cefacc2946cc0726a480533/services/sso_lifecycle.go#L79) and [L116](https://github.com/cloudfoundry/cf-acceptance-tests/blob/ee8e82d7199fa2726cefacc2946cc0726a480533/services/sso_lifecycle.go#L116).

`apiEndpoint` was previously assigned on [L43](https://github.com/cloudfoundry/cf-acceptance-tests/blob/145a74556952c17cbc898bfb7147dd3fc2b198a1/services/sso_lifecycle.go#L43-L44), and was there because its value was consumed by immediately after that on L44 by `SetOauthEndpoints`. That's no longer the case.

`SetOauthEndpoints` now takes an `apiInfoEndpoint` instead (the domain retrieved from the new [GetApiInfoUrl](https://github.com/sap-contributions/cf-acceptance-tests/blob/ee8e82d7199fa2726cefacc2946cc0726a480533/helpers/services/broker.go#L127-L137)). I guess I felt it made sense to assign `apiEndpoint` elsewhere to avoid giving the impression there's still some connection between the two. 

Since `apiEndpoint` is just a concatenation of a couple of values from config it could, of course, be assigned anywhere in `BeforeEach`. Happy to move it if you still feel it ought to be somewhere else, but hopefully this explains it.

Reply:
Gotcha, thanks for the clarification!